src/main/java/com/gitblit/wicket/GitBlitWebApp.properties

```diff
@@ -48,6 +48,7 @@ gb.password = password
 gb.tagger = tagger
 gb.moreHistory = more history...
 gb.difftocurrent = diff to current
+gb.allrepos = in all repositories
```

Comment:
Let's tweak this to "All Repositories" so that it can be used elsewhere in the app, if needed.